internal/ui/main_page.go

```diff
@@ -0,0 +1,53 @@
+package ui
+
+import (
+	"github.com/rivo/tview"
+)
+
+func MainPage(app *tview.Application) error {
+	layoutListDownload := tview.NewFlex().
+		SetDirection(tview.FlexRow)
+
+	txtDownloads := tview.NewTextView().
+		SetText("Downloads").
+		SetTextAlign(tview.AlignLeft)
+
+	listDownload := tview.NewList().
+		AddItem("123456781", "Replay 1", '1', nil).
+		AddItem("123456781", "Replay 2", '2', nil)
```

Comment:
[this comment](https://github.com/ronaudinho/drt/pull/13/files#r1194546190) also applies for `selected func()` in  `AddItem` here

Reply:
what if I leave a `// TODO` comment in this line? then we implement it in the next PR?